test/tasks/build.test.js

```diff
@@ -164,10 +161,7 @@ describe('Build task', function() {
 		});
 
 		afterEach(function() {
-			if (fs.existsSync('build/main.css')) {
-				fs.unlink('build/main.css');
-				fs.rmdir('build');
-			}
+			return exec('rm -rf build');
```

Comment:
I don't mind having denodeify or rimraf as a dependency, but I do think it makes sense to use the same one every time.